docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +379,285 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
+
+We currently recommend using the snippet id for any system which _might maybe_ end up with multiple snippets.
+
+## Ingestion
+
+There are two APIs with which you can ingest documents in a way where it can
+produce multiple snippets:
+
+- using the [`split` option with `snippet`](#automatic-document-splitting)
+- using the [`split` option with `file`](#file-upload)
+
+Be aware that you need to use sufficient long text for it to be split.
+
+You could use the example from the [`split` option section](#automatic-document-splitting) or use
+your own document, something along ~1000 words will likely generate multiple splits.
+
+## Semantic Search
+
+As mentioned the search results are based on snippets not documents, as such a search
+can yield multiple different snippets of the same document.
+
+The following is an example that performs a search for a specific snippet.
+
+```bash
+curl -X POST "$URL/semantic_search" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "document": { "id": { "document_id": "effaffe", "sub_id": 2 },
+        "include_properties": true
+    }'
+```
+
+This will return the snippets of which the semantic representation is closest to the
+semantic representation of snippet with the `sub_id: 2` of the given document.
+
+```{note}
+Currently, the whole document used for the search is excluded from the search results, so, the result will not contain snippets from the same document.
+```
+
+For personalize semantic search with provided history snippet ids can also be used instead of
+document ids:
+
+```json
+"personalize": {
+    "exclude_seen": true,
+    "user": {
+         "history": [
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 0 },
+                 "timestamp": "2000-05-14T20:22:50Z"
+             },
+             {
+                 "id": { "document_id": "valid_doc_id1", "sub_id": 1 },
+                 "timestamp": "2000-05-15T20:22:50Z"
+             }
+         ]
+    }
+}
+```
+
+When the history contains an interaction with a document which has multiple snippets, it will be
+processed similar to registering an interaction with each snippet of that document.
+
+`exclude_seen` always refers to whole documents.
+In the example above, even if there were a snippet with id `{ "document_id": "valid_doc_id1", "sub_id": 30 }` it would exclude the whole document `"valid_doc_id1"` no matter how many snippets it has.
+
+Please be aware that using semantic search based on a document id of a document with multiple snippets has some technical limitations. Mainly instead of searching based on the document as a whole it might search based on
+a single snippet of the document. This will change in the future but until then we recommend to always use the snippet id for now.
```

Comment:
```suggestion
Please be aware that using semantic search based on a document id of a document with multiple snippets has some limitations. Mainly, instead of searching based on the document as a whole, it searches based on
a single snippet of the document. For this reason, we recommend to always use the snippet id for now.
```